### PR TITLE
Should not have added discord deps to dev

### PR DESCRIPTION
**Pull request by rm-you**
Also fix a discord.js change to the SlashCommand type

### CODE REVIEW

File: src/types.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {
-	SlashCommandBuilder,
+	SlashCommandOptionsOnlyBuilder,
```
Comment (rm-you) [inline src/types.d.ts:2]: I guess this changed in discordjs 14.15.x?

File: package.json [hunk 2]
```diff
@@ -51,7 +51,6 @@
     "@types/node-fetch": "^2.6.6",
     "@types/node-forge": "^1.3.7",
     "@typescript-eslint/eslint-plugin": "^6.6.0",
-    "discord.js": "^14.15.3",
```
Comment (rm-you) [inline package.json:54]: This can't be here or it will prune discord.js during the image creation XD

File: package-lock.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tunnel-quest-bot",
   "version": "1.1.2",
-  "lockfileVersion": 2,
```
Comment (rm-you) [inline package-lock.json:4]: I just regenerated the entire package-lock file for the first time in a while, and this is what it did -- I guess the new version doesn't require the dependencies section? Or was that something we added manually?

File: package.json (hunk 2 above)
Comment (jamesjamail) [reply package.json:54]: weird- idk how discordjs got added as a dev dep

**Review by jamesjamail — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (rm-you) [reply package.json:54]: Me, I added it by accident 😂

**Pull request merged**